<launch>
    <arg name='gui' default='true' />

    <include file='$(find gazebo_ros)/launch/empty_world.launch'>
      <arg name='world_name' value='$(find a3c)/worlds/simple_maze.world'/>
      <arg name='gui' value='$(arg gui)'/>
    </include>


     <!--Spawn simulated quadrotor uav-->
    <include file='$(find cvg_sim_gazebo)/launch/spawn_quadrotor.launch' >
      <arg name='model' value='$(find ardrone_sim)/urdf/quadrotor_sensors.urdf.xacro'/>
    </include>

    <arg name='model' default='$(find ardrone_sim)/urdf/quadrotor.urdf.xacro'/>
    <arg name='restore' default='False' />

    <arg name='worker-args' default='' />
    <param name='controller_rate' type='double' value='10.0'/>
    <param name='num_images' type='int' value='4'/>
    <param name='image_scale' type='double' value='1'/>
    <param name='images_to_skip' type='int' value='0'/>
    <param name='restore' type='bool' value='$(arg restore)'/>

    <!--<node name='agent_node'-->
      <!--pkg='a3c'-->
      <!--type='random_agent.py'-->
      <!--output='screen'-->
      <!--required='true'-->
    <!--/>-->

    <node name='agent_node'
      pkg='a3c'
      type='job.py'
      output='screen'
      required='true'
      args='$(arg worker-args)'
    />

</launch>
